src/Stripe.net/Constants/Events.cs

```diff
@@ -504,11 +639,47 @@ public static class Events
         /// </summary>
         public static string TaxRateUpdated => "tax_rate.updated";
 
+        /// <summary>
+        /// Occurs whenever a top-up is canceled.
+        /// </summary>
+        public static string TopupCanceled => "topup.canceled";
+
+        /// <summary>
+        /// Occurs whenever a top-up is created.
+        /// </summary>
+        public static string TopupCreated => "topup.created";
+
+        /// <summary>
+        /// Occurs whenever a top-up fails.
+        /// </summary>
+        public static string TopupFailed => "topup.failed";
+
+        /// <summary>
+        /// Occurs whenever a top-up is reversed.
+        /// </summary>
+        public static string TopupReversed => "topup.reversed";
+
+        /// <summary>
+        /// Occurs whenever a top-up succeeds.
+        /// </summary>
+        public static string TopupSucceeded => "topup.succeeded";
+
         /// <summary>
         /// Occurs whenever a new transfer is created.
         /// </summary>
         public static string TransferCreated => "transfer.created";
 
+        /// <summary>
+        /// Occurs whenever a transfer failed.
+        /// </summary>
+        public static string TransferFailed => "transfer.failed";
```

Comment:
This is incorrect, it's a legacy event and should not be here since we're pinned to the most recent API version